accounts-bench/Cargo.toml

```diff
@@ -14,7 +14,7 @@ log = { workspace = true }
 rayon = { workspace = true }
 solana-logger = { workspace = true }
 solana-measure = { workspace = true }
-solana-runtime = { workspace = true }
+solana-runtime = { workspace = true, features = ["dev-context-only-utils"] }
```

Comment:
hey, thanks for trying dcou. :)

as ci is failing, you need to add this crate to the whitelist like this: https://github.com/solana-labs/solana/pull/32508#discussion_r1285149559